.fixtures.yml

```diff
@@ -1,4 +1,8 @@
 fixtures:
+  forge_modules:
+    stdlib:
+      repo: "puppetlabs/stdlib"
+      ref: "4.6.0"  
```

Comment:
I'll change it to 4.0.0, as that is the first version where str2bool accepts booleans also.